plugins/org.python.pydev/tests_completions/org/python/pydev/ast/codecompletion/PythonCompletionWithBuiltinsPython3Test.java

```diff
@@ -479,4 +479,19 @@ public void testContextManagerCompletion8() throws Exception {
         requestCompl(s, s.length(), -1, new String[] { "denominator" });
     }
 
+    public void testSubclassWithGenericTypes() throws Exception {
```

Comment:
Please create a new case with:

```
import typing

class SubClass(typing.TextIO):
    def some_method(self) -> str:
        pass

c = SubClass()
c.
```

Where the completions must be at least `some_method()` as well as `close()`

-- i.e.: `close` must be gotten from the `typing.TextIO` because it inherits `typing.IO`.